test/modular/Mocha6.js

```diff
@@ -40,12 +40,13 @@ export class Mocha6 extends Mocha {
 
   importFiles() {
     var promiseImports = this.files.map((file) => {
-      file = './' + file.replace(/\\/g, '/');;
-      this.suite.emit('pre-require', global, file, this);
+      file = './' + file.replace(/\\/g, '/');
+      // TODO(arv): Why do we need to use global here?
+      this.suite.emit('pre-require', Reflect.global, file, this);
```

Comment:
Mocha uses this event as a trigger to attach the definitions of suite/before/after to the argument.  That's how our older tests get a definition for suite().   The es6-based tests should always be using the imported suite().  But the feature tests are not converted. 

(Mocha does this to allow different "UI" versions to be mixed I guess).

What problem did you encounter. This was all working for me.